apps/main/src/llamalend/features/manage-loan/hooks/useRemoveCollateralForm.ts

```diff
@@ -133,16 +133,18 @@ export const useRemoveCollateralForm = <
 
   const expectedCollateral = useMemo(
     () =>
-      mapQuery(userState, (state) => {
-        // An error will be thrown by the validation suite, this is just for preventing negative collateral in the UI
-        const value =
-          state?.collateral != null &&
-          values.userCollateral != null &&
-          decimal(
-            BigNumber.max(new BigNumber(state.collateral).minus(new BigNumber(values.userCollateral)), '0').toString(),
-          )
-        return value ? { value, tokenSymbol: collateralToken?.symbol } : null
-      }),
+      // An error will be thrown by the validation suite, the "max" is just for preventing negative collateral in the UI
+      mapQuery(
+        userState,
+        (state) =>
+          state.collateral &&
+          values.userCollateral && {
+            value: decimal(
+              BigNumber.max('0', new BigNumber(state.collateral).minus(new BigNumber(values.userCollateral))),
```

Comment:
```suggestion
              BigNumber.max(0, new BigNumber(state.collateral).minus(new BigNumber(values.userCollateral))),
```

Reply:
btw I also found the [clamp function](https://github.com/curvefi/curve-frontend/blob/a25582753192521d06ef6d8a2320fba83bd93f21/packages/curve-ui-kit/src/shared/ui/NumericTextField.tsx#L73) which could be adapter to `Decimal` clamps